.gitignore

```diff
@@ -46,3 +46,9 @@ README.html
 .mypy_cache
 .cache
 .konchrc.local
+
+# Visual Studio Code
+.vscode/
```

Comment:
This belongs in your [global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Thanks for the info, very helpful.